mysql-test/suite/binlog_encryption/t/binlog_encryption_without_keyring.test

```diff
@@ -0,0 +1,37 @@
+# Bug: PS-288: mysqld got signal 6 if you try to enable encrypt_binlog + binary log without keyring-file plugin
+
+# Server is aborted when binlog_error_action is set to ABORT_SERVER and binlog encryption could not be
+# initialized. An error message generated in server log got improved in this case by mentioning that
+# encryption could be a reason why server got aborted.
+
```

Comment:
`--source include/not_embedded.inc` as this test restarts the server

Reply:
Done